### PR TITLE
Add some suggestions and fix spacing

### PR DESCRIPTION
**Pull request by nirinchev**

### CODE REVIEW

File: Managers/BlackJackGameManager.cs [hunk 1]
```diff
@@ -35,23 +26,25 @@ public BlackJackGameManager(BlackJackHumanPlayer human, BlackJackComputerPlayer
...
+			if (Human.Score > 21)
+			{
 				GameWinner("You lose");
 			}
 			else if (Human.Score == 21)
 			{
 				GameWinner("You win");
 			}
-		    return hitted;
+			return hitted;
```
Comment (nirinchev) [inline Managers/BlackJackGameManager.cs]: This is example line comment. Also, the past tense of ```hit``` is ```hit```, not ```hitted``` :D

File: Managers/BlackJackGameManager.cs [hunk 2]
```diff
@@ -91,15 +86,18 @@ public void Stand()
 			}
 		}
 
-		private void GameWinner(string winnerText){
+		// This should be a ver
```
Comment (nirinchev) [inline Managers/BlackJackGameManager.cs]: *verb

**Comment by nirinchev** [comment]
I'd suggest that you guys install the [ProductivityPowerTools](https://visualstudiogallery.msdn.microsoft.com/34ebc6a2-2777-421d-8914-e29c1dfa7f5d) extension and configure it to format the document on save - it will clean up and reformat your files automatically.

**Comment by mihailirinchev** [comment]
We are using Visual Studio 2013, so the correct link should be [this](https://visualstudiogallery.msdn.microsoft.com/dbcb8670-889e-4a54-a226-a48a15e4cace)

**Pull request merged**